tests/scripts/remote_monitoring_tests/helpers/list_reco_json_schema.py

```diff
@@ -395,6 +479,174 @@
+                                                },
+                                                "required": [
+                                                  "amount",
+                                                  "format"
+                                                ]
+                                              }
+                                            },
+                                            "required": [
+
+                                            ]
+                                          },
+                                          "requests": {
+                                            "type": "object",
+                                            "properties": {
+                                              "memory": {
+                                                "type": "object",
+                                                "properties": {
+                                                  "amount": {
+                                                    "type": "number"
+                                                  },
+                                                  "format": {
+                                                    "type": "string"
+                                                  }
+                                                },
+                                                "required": [
+                                                  "amount",
+                                                  "format"
+                                                ]
+                                              },
+                                              "cpu": {
+                                                "type": "object",
+                                                "properties": {
+                                                  "amount": {
+                                                    "type": "number"
+                                                  },
+                                                  "format": {
+                                                    "type": "string"
+                                                  }
+                                                },
+                                                "required": [
+                                                  "amount",
+                                                  "format"
+                                                ]
+                                              }
+                                            },
+                                            "required": [
+
+                                            ]
+                                          }
+                                        },
+                                        "required": [
+
+                                        ]
+                                      },
+                                      "current": {
```

Comment:
@bharathappali - Current section is repeated again?

Reply:
Removed the duplicate entries. Thanks for review @chandrams